src/labels/CustomLabels.labels

```diff
@@ -4214,6 +4270,548 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
+        <fullName>geHeaderError</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error</shortDescription>
+        <value>Error</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderFormFieldsDefaultSectionName</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Name of default form section</shortDescription>
+        <value>Gift Entry Form</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderFormFieldsLeftCol</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header for Form Fields sidebar</shortDescription>
+        <value>Form Fields &amp; Field Bundles</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderFormFieldsModalDeleteSection</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header for deleting section in settings modal</shortDescription>
+        <value>Delete Section</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderFormFieldsModalRenameSection</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header for renaming section in settings modal</shortDescription>
+        <value>Rename Section</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderFormFieldsModalSectionSettings</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header for form fields section settings modal</shortDescription>
+        <value>Section Settings</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderFormFieldsRightCol</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header for Form Fields body</shortDescription>
+        <value>Configure Form Fields</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderNewTemplate</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Template header</shortDescription>
+        <value>New Template</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderTemplateInfoLeftCol</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header for gift entry template information</shortDescription>
+        <value>Gift Entry Template Information</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderWarning</fullName>
```

Comment:
@psadie-sfdo and @markross-sfdo should this dev name be more generalized?

Reply:
Makes sense to me. Updating!